models/comments-model.js

```diff
@@ -10,3 +10,25 @@ exports.selectCommentsByArticleID = (article_id) => {
       return data.rows;
     });
 };
+
+exports.insertComment = (article_id, { username, body }) => {
+  return db
```

Comment:
Technically this is ok, but you would be better to refactor to a reusable function like the one in the advanced error handling notes.